packages/access-client/src/types.ts

```diff
@@ -80,8 +80,8 @@ export interface AgentData<T> {
   meta: AgentMeta
   principal: T
   currentSpace?: DID
-  spaces: Map<DID, SpaceMeta>
-  delegations: Map<CIDString, { meta?: DelegationMeta; delegation: Delegation }>
+  spaces: Map<DID | undefined, SpaceMeta>
```

Comment:
No...?

Reply:
just a workaround for TS, sometimes you want to `spaces.get(didThatMayBeUndefined)` and you can't because the type is too strict